pkg/pillar/cmd/logmanager/logmanager.go

```diff
@@ -322,7 +322,7 @@ func Run(ps *pubsub.PubSub) {
 				warningTime, errorTime)
 
 		case change := <-deferredChan:
-			_, err := devicenetwork.VerifyDeviceNetworkStatus(*deviceNetworkStatus, 1, 15)
+			_, err := devicenetwork.VerifyDeviceNetworkStatus(deviceNetworkStatus, 1, 15)
```

Comment:
See suggestion below in a different comment.

In this case you do not clear out any per interface errors in deviceNetworkStatus, which is potentially problematic. Not that logmanager doesn't care about that; it just wants to know whether it should try to send again. Hence the per-interface errors is a distraction for it.